src/screens/App.tsx

```diff
@@ -1,23 +1,95 @@
-import { useState } from "react";
-
+import {
+  ButtonLogin,
+  ButtonCreate,
+  ButtonRegister
+} from "../components/Button/Button";
+import { FeatureTitle, FeatureDescription } from "../components/Text/Text";
 export function App() {
-  const [count, setCount] = useState(0);
-
   return (
-    <main className="flex h-screen w-full flex-col items-center justify-center gap-2 bg-black">
-      <h1 className="text-xl text-white">
-        Vite + React + Typescript + TailwindCSS | Hi
-      </h1>
-      <button
-        type="button"
-        className="bg-white p-2 text-black"
-        onClick={() => {
-          setCount((count) => count + 1);
-        }}
-      >
-        Increase
-      </button>
-      <p className="text-white">Count: {count}</p>
+    <main className="h-screen w-full">
+      <div className="min-h-1/2 bg-blue-600 pb-16">
+        <div className=" mx-auto max-w-screen-2xl">
+          <div className="left-4 top-4 flex items-center font-bold text-white sm:left-8 md:left-12 lg:left-16 xl:left-20">
+            <img
+              src="/Logos/logo.png"
+              alt=""
+              className="mr-2 h-16 w-16 rounded-md border border-gray-500"
+            />
+            <p className="ml-2 text-2xl">CapyFile</p>
+            <div className="flex flex-grow justify-end gap-4">
+              <ButtonLogin text="Login" type="anchor" />
+              <ButtonRegister text="Register" type="anchor" />
+            </div>
+          </div>
+
+          <div className=" top-38 left-4 m-4 mt-16 font-bold text-white sm:left-8 md:left-12 lg:left-16 xl:left-20">
+            <h1 className="text-5xl">
+              Access your files
+              <br />
+              everywhere
+            </h1>
+            <div className="mt-14">
+              <h1 className="text-2xl">
+                Effortlessly store, organize and access your
+                <br /> files with built-in redundancy.
+              </h1>
+            </div>
+            <div className="mt-16">
+              <ButtonCreate
+                text="Create an Account"
+                type="anchor"
+                className="mt-2 "
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className="min-h-1/2 relative bg-white">
```

Comment:
1. The `relative` position isn't needed. 
2. The `<div>` can be replaced by a `<section>`